mixxx-now-playing.sh

```diff
@@ -10,37 +10,73 @@ OS=`uname`
 
 echo "Detected OS: $OS"
 
-while true; do
-	while pgrep -i mixxx > /dev/null; do
-	
+quartzListAllWindows=$'import Quartz\nprint(Quartz.CGWindowListCopyWindowInfo(Quartz.kCGWindowListExcludeDesktopElements|Quartz.kCGWindowListOptionAll,Quartz.kCGNullWindowID))'
+
+lastKnownTitle=" "
+
+FormatAndWriteTitle(){
+	currentTitle=$(  
+	echo "$1" |
```

Comment:
this echo breaks the song title resulting in no output.

Reply:
I've changed how this works, now using a regex to extract the track title, but it still uses an echo to feed the pipe. It works fine on my GNU Bash, what environment are you running it in?